controllers/buildpipeline/buildpipeline_adapter_test.go

```diff
@@ -760,7 +760,7 @@ var _ = Describe("Pipeline Adapter", Ordered, func() {
 		})
 
 		It("Can add an annotation to the build pipelinerun", func() {
-			pipelineRun, err := adapter.annotateBuildPipelineRun(buildPipelineRun, "test", "value")
+			pipelineRun, err := tekton.AnnotateBuildPipelineRun(buildPipelineRun, "test", "value", adapter.client, adapter.context)
```

Comment:
Context should be the first parameter

https://pkg.go.dev/context

> The Context should be the first parameter, typically named ctx:

Reply:
Switched.

Reply:
"First", like absolutely first, not relatively first

Reply:
Sorry, updated.